src/assertions.jl

```diff
@@ -7,3 +7,8 @@ function assert_continuous(table)
   types = schema(table).scitypes
   @assert all(T <: Continuous for T in types) "columns must hold continuous variables"
 end
+
+# assert that column is categorical
+function assert_categorical(x)
+  @assert x isa CategoricalArray "The selected column must be cetegorical."
```

Comment:
We need to use the scitype of the column instead. Check the assert_continuous for an idea. The return of the function will be something like Categorical. Check the type hierarchy of ScientificTypes.jl to learn more.